### PR TITLE
Add options for highlightjs to be passed in from the .use call

### PR DESCRIPTION
**Pull request by ericcirone**
This will allow users to add in options to configure highlightjs

### CODE REVIEW

**Comment by esbeto** [comment]
From the documentation:

https://highlightjs.readthedocs.io/en/latest/api.html?highlight=configure

## `configure(options)`
### Configures global options:

- `tabReplace`: a string used to replace TAB characters in indentation.
- `useBR`: a flag to generate `<br>` tags instead of new-line characters in the output, useful when code is marked up using a non-`<pre>` container.
- `classPrefix`: a string prefix added before class names in the generated markup, used for backwards compatibility with stylesheets.
- `languages`: an array of language names and aliases restricting auto detection to only these languages.

Accepts an object representing options with the values to updated. Other options don’t change

```
hljs.configure({
  tabReplace: '    ', // 4 spaces
  classPrefix: ''     // don't append class prefix
                      // … other options aren't changed
})
hljs.initHighlighting();
```

**Comment by esbeto** [comment]
So this might help close https://github.com/metachris/vue-highlightjs/issues/15 and https://github.com/metachris/vue-highlightjs/issues/10

File: index.js [hunk 1]
```diff
@@ -4,7 +4,8 @@ var hljs = require('highlight.js');
 // import hljs from 'highlight.js';
 
 var vueHighlightJS = {};
-vueHighlightJS.install = function install(Vue) {
+vueHighlightJS.install = function install(Vue, options) {
```
Comment (esbeto) [inline index.js:7]: Might need to add a default value, or handle the possibility of options not being sent.
```js
vueHighlightJS.install = function install(Vue, options = {}) {
```

**Comment by onekiloparsec** [comment]
Any idea when this would be merged ? #10 is a rather pressing problem... Thanks!

**Comment by metachris** [comment]
Sorry for the delay, and thanks for the contribution!

File: index.js (hunk 1 above)
Comment (metachris) [reply index.js:7]: This would be good to add!

**Comment by metachris** [comment]
Closing this issue because this project is officially superseded by https://github.com/highlightjs/vue-plugin

Thanks for participating <3

**Pull request closed without merging**